recipes/libwebsockets/all/conanfile.py

```diff
@@ -236,11 +236,8 @@ def validate(self):
         if self.options.shared and self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "5":
             # https://github.com/conan-io/conan-center-index/pull/5321#issuecomment-826367276
             raise ConanInvalidConfiguration("{}/{} shared=True with gcc<5 does not build. Please submit a PR with a fix.".format(self.name, self.version))
-        if Version(self.version) <= "4.0.15" and self.settings.compiler == "apple-clang" and Version(self.settings.compiler.version) >= "12":
-            raise ConanInvalidConfiguration("{}/{} with apple-clang>=12 does not build. Please submit a PR with a fix.".format(self.name, self.version))
         if Version(self.version) >= "4.3.2":
-            if ("Visual" in str(self.settings.compiler.version) and Version(self.settings.compiler.version) < 16) or  \
-                    ("msvc" == str(self.settings.compiler.version) and Version(self.settings.compiler.version) < 192):
```

Comment:
I think this is a logic change.  I think it should also check that the compiler is msvc? I'm getting validation fails here when building with gcc 13.2.0
``` 
libwebsockets/4.3.3: Invalid: libwebsockets/4.3.3 requires at least Visual Studio 2019
````

Reply:
Actually shouldn't it be `not` ?

Reply:
Woopsie, all configurations are invalid indeed. Don't hesitate to make a PR fixing the error